docs/SUMMARY.md

```diff
@@ -0,0 +1,34 @@
+* [Home](index.md)
+* [Installation](installation.md)
+* Tutorial
+    * [Introduction to Ibis](tutorial/01-Introduction-to-Ibis.ipynb)
+    * [Aggregating and Joining](tutorial/02-Aggregates-Joins.ipynb)
+    * [Lazy Mode and Logging](tutorial/03-Expressions-Lazy-Mode-Logging.ipynb)
+    * [More Value Expressions](tutorial/04-More-Value-Expressions.ipynb)
+    * [Creating and Inserting External Data](tutorial/05-IO-Create-Insert-External-Data.ipynb)
+    * Advanced Topics
+        * [Complex Filtering](tutorial/06-ComplexFiltering.ipynb)
+        * [Analytics Tools](tutorial/07-Analytics-Tools.ipynb)
+        * [Geospatial Analysis](tutorial/08-Geospatial-Analysis.ipynb)
+* [Ibis for SQL Programmers](ibis-for-sql-programmers.ipynb)
+* [User Guide](user_guide/)
+* [Backends](backends/)
+* [Contribute](contribute/)
+* Community
+    * [About](about/)
+    * [Ask a question (StackOverflow)](https://stackoverflow.com/questions/tagged/ibis)
+    * [Chat (Gitter)](https://gitter.im/ibis-dev/Lobby)
```

Comment:
Do we still have the gitter channel? 
We could/should promote the github discussions channel as well.

Reply:
We definitely have the gitter channel, and it's seeing activity on the order of a few messages per week.